src/main/java/seedu/cmdo/logic/parser/MainParser.java

```diff
@@ -125,6 +125,20 @@ private Command prepareAdd(String args){
         String[] splittedArgs = getCleanString(args).split(" ");
         reducedArgs = extractDueByDateAndTime(args);
         LocalDateTime dt;
+        
+        //check for empty add
+        if(args.isEmpty()){
+            return new IncorrectCommand(
+                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_USAGE));
+        }
+        
+        // check for add 1-23
```

Comment:
Ok, works as intended. But here's the issue. `add 23 new things` gives us a Task with detail "new things" at 11pm. Possible to read the 23 only if there is an `at`?